contracts/Strategy.sol

```diff
@@ -148,106 +138,45 @@ contract Strategy is BaseStrategy {
         isWantIncentivised = _isWantIncentivised;
         isInvestmentTokenIncentivised = _isInvestmentTokenIncentivised;
         leaveDebtBehind = _leaveDebtBehind;
-        require(maxLoss <= 10_000);
```

Comment:
any good reason to delete this?

Reply:
This was probably removed when contract size was near the limit, before the refactor to create a separate lib. Fixed in c92f32ca98057c56d99a88c2d78814416b95daf4.